frontend-react/src/hooks/UseLookupTable.ts

```diff
@@ -10,9 +10,15 @@ import {
 } from "../network/api/LookupTableApi";
 import { showError } from "../components/AlertNotifications";
 
+export interface TableAttributes {
+    version: number;
+    createdAt: string;
+    createdBy: string;
+}
+
 export async function getLatestVersion(
     tableName: LookupTables
-): Promise<number> {
+): Promise<TableAttributes | null> {
```

Comment:
since this isn't just returning a version number anymore, i figured `null` would be a better indication of failure than `-1`